whois-api/src/main/java/net/ripe/db/whois/api/httpserver/JettyBootstrap.java

```diff
@@ -137,8 +142,10 @@ public JettyBootstrap(final RemoteAddressFilter remoteAddressFilter,
         this.trustedIpRanges = trustedIpRanges;
         this.rewriteEngineEnabled = rewriteEngineEnabled;
         LOGGER.info("Rewrite engine is {}abled", rewriteEngineEnabled ? "en" : "dis");
-        this.dosFilterMBeanName = ObjectName.getInstance("net.ripe.db.whois:name=DosFilter");
+        this.dosLookUpFilterMBeanName = ObjectName.getInstance("net.ripe.db.whois:name=DosFilter");
```

Comment:
Why are there two separate management beans? Can't you create *one* to maintain the whitelist, which is shared between Lookup and Update?

Reply:
I did refactor this. We don't need to create any bean here